content/bash.rst

```diff
@@ -52,10 +52,9 @@ General
 
 * Always guard your Bash scripts from unexpected errors by using
 
-.. sourcecode:: bash
-
+```bash
```

Comment:
No, I mistakenly assumed '.rst' files were as the '.md' files but for git websites, as ReStructured Text shares similarities with Markdown. I will quickly search and correct all the occurrences with the original .. sourcecode::.
Learnt a new language I guess :P

I found an online rst tester at [rst.ninjs.org](http://rst.ninjs.org) so I will compare with that, my comparison with Markdown was wrong.